pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExporter.java

```diff
@@ -63,11 +65,14 @@
     private final String customGuid;
     private final String customActivity;
     private final RestHighLevelClient elasticsearchClient;
+    private final AddressService addressService;
+
 
     CustomExporter(Config mainConfig, Config exportConfig) {
         this.exportConfig = exportConfig;
         this.customActivity = exportConfig.getString(CustomExportConfigFile.ACTIVITY);
         this.customGuid = exportConfig.getString(CustomExportConfigFile.STUDY_GUID);
+        addressService = new AddressService();
```

Comment:
Doesn't look like we use `AddressService` here?

Reply:
Removed